picard/formats/id3.py

```diff
@@ -459,6 +476,21 @@ def _save(self, filename, metadata):
                     desc = ''
                 for value in values:
                     tags.add(id3.USLT(encoding=encoding, desc=desc, text=value))
+            elif name.startswith('syncedlyrics:') or name == 'syncedlyrics':
+                if ':' in name:
+                    desc = name.split(':', 1)[1]
+                else:
+                    desc = ''
+                for value in values:
+
+                    sylt_lyrics = []
+                    timestamp_and_lyrics = batched(re.split(self.__lrc_re_parse, value)[1:], 2)
+                    for timestamp, lyrics in timestamp_and_lyrics:
+                        minutes, seconds, hundredths = timestamp[1:-1].replace(".", ":").split(':')
+                        milliseconds = int(minutes) * 60 * 1000 + int(seconds) * 1000 + int(hundredths) * 10
+                        sylt_lyrics.append((lyrics, milliseconds))
```

Comment:
Same here, better move this code to a separate small method, and write tests for it.